packages/fxa-settings/src/components/Settings/FlowSetupRecoveryPhoneSubmitNumber/index.tsx

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import FlowContainer from '../FlowContainer';
+import ProgressBar from '../ProgressBar';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import Banner from '../../Banner';
+import { BackupRecoveryPhoneImage } from '../../images';
+import { getLocalizedErrorMessage } from '../../../lib/error-utils';
+import { useFtlMsgResolver } from '../../../models';
+import FormPhoneNumber from '../../FormPhoneNumber';
+
+export type FlowSetupRecoveryPhoneSubmitNumberProps = {
+  localizedBackButtonTitle: string;
+  localizedPageTitle: string;
+  navigateBackward: () => void;
+  navigateForward: () => void;
+  verifyPhoneNumber: (phoneNumber: string) => Promise<void>;
+};
+
+export const FlowSetupRecoveryPhoneSubmitNumber = ({
+  localizedBackButtonTitle,
+  localizedPageTitle,
+  navigateBackward,
+  navigateForward,
+  verifyPhoneNumber,
+}: FlowSetupRecoveryPhoneSubmitNumberProps) => {
+  const [localizedErrorBannerMessage, setLocalizedErrorBannerMessage] =
+    useState('');
+
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const clearBanners = async () => {
+    return new Promise<void>((resolve) => {
+      setLocalizedErrorBannerMessage(''); // Clear the banner message
+      setTimeout(() => resolve(), 0); // Wait for DOM update
```

Comment:
Hm why do we need a `setTimeout`? I think this isn't great for performance. In other places, we simply clear it with `''`.

Reply:
I was attempting to fix an issue with the banner message not always being announced or getting interrupted - unfortunately this wasn't 100% reliable so I will remove.